types/jsonwebtoken/index.d.ts

```diff
@@ -59,7 +59,7 @@ export interface SignOptions {
 
 export interface VerifyOptions {
     algorithms?: string[];
-    audience?: string | string[];
+    audience?: string | RegExp | ReadonlyArray<string | RegExp>;
```

Comment:
I don't see anything that marks it strictly as `ReadonlyArray`, should stick with just `Array`.